.github/workflows/rebase-cmd-dispatch.yml

```diff
@@ -0,0 +1,15 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
```

Comment:
can we only run this action if the text == `/rebase`? otherwise we'll get a lot of actions which do nothing

Reply:
AFAIK, there is no way to trigger workflow based on the content of the issue comment. Eventually we could add a condition for job like `if: contains(github.event.comment.body, '/rebase')` so that it will be fast-skipped but still the Action space will be polluted with tons of skipped actions triggered, each one triggered by a single comment.

Reply:
I guess fast-skip is better than nothing, let's try and we'll se if it works fine for us

Reply:
Added.